plone/scale/storage.py

```diff
@@ -229,7 +229,8 @@ def pre_scale(self, **parameters):
             # There is nothing we can do.
             return
         fieldname = parameters.get("fieldname", None)
-        if scaling_factory.get_original_value(fieldname=fieldname) is None:
+        field = scaling_factory.get_original_value(fieldname=fieldname)
```

Comment:
I was a bit confused when reading the `pre_scale` function last week since this variable contains a field value, not a field. Maybe it should be called `value` instead of `field`

Reply:
Fair point. Fixed in commit c4ceb413cba4d3d1fb786204d5219213c40dd2e4.